### DISCUSSION TITLE
Multihead finetune pt and default models

### DISCUSSION THREAD

**Discussion by marcob3** [ACEsuit/mace#711] [Q&A] [answered]
Greetings!

I am Marco Bertani, from University of Modena (Italy). I am trying to finetune the mace_agnesi_medium.model using the multihead approach. 
It generates a pt_head and default models. I do not understand the difference between the two models and which one I should use for running MD after the training.

I appreciate your help.
Marco

**Comment by ilyes319** [comment, marked as answer]
Hey, the pt_head corresponds to the head for the pretraining replay set. This head is there to ensure that the model does not forget too much during finetuning. The default head is the head that is used for your own training set. 
When you do MD you do not have to pick a head, it will automatically use the one from your data.
If you want to select another head, you can do atoms.info["head"] = "pt_head", but I advise you not to do that.

**Reply by marcob3** [reply]
Thank you very much for your reply. 
So also when I convert the model for lammps it automatically uses the default one?

**Reply by ilyes319** [reply]
Yes, that is handled automatically.

**Comment by wuziqiqiqi** [comment]
Hello, guys! I have the same question, and thanks a lot for the explanation! Could you explain a bit more about on _"This head is there to ensure that the model does not forget too much during finetuning."_ ? i.e. how does it help? what kind of number should I expect to see? any literature I can look into? Thanks a lot in advance!

**Reply by gabor1** [reply]
It helps because through that head, the model is made to reproduce the old data that was used to pretrain it. so it restraints the model weights to not deviate too far from the pretraining weights.
